lua/astrocommunity/pack/java/README.md

```diff
@@ -11,6 +11,32 @@ This plugin pack does the following:
 - Adds `jdtls` language server
 - Adds `lemminx` language server (XML)
 - Adds [nvim-jdtls](https://github.com/mfussenegger/nvim-jdtls) for language specific tooling
-    - Enables [hot reloading](https://github.com/mfussenegger/nvim-jdtls/issues/80)
+  - Enables [hot reloading](https://github.com/mfussenegger/nvim-jdtls/issues/80)
 - Adds `clang_format` through null-ls
-- Adds `javadbg` and `javatest` debug adapters for nvim-dap 
+- Adds `javadbg` and `javatest` debug adapters for nvim-dap
+
+## Tips
+
+`jdtls` requires Java 11+. If your project requires you to use Java 8, you could try and set the java runtime for `jdtls`. Something along the lines of:
```

Comment:
```suggestion
`jdtls` requires Java 11+ but can be used to develop on any Java version. If you develop  using different Java runtimes, you can set the runtimes  you have available in the settings of `jdtls`. Here is a simple example:
```

This is a bit more clear since the code shown is unrelated to Java 8